packages/devtools-node/src/fs.ts

```diff
@@ -22,7 +22,7 @@ export async function createComposite(
   ceramic: CeramicClient | string,
   path: PathInput
 ): Promise<Composite> {
-  const client = ceramic instanceof CeramicClient ? ceramic : new CeramicClient(ceramic)
+  const client = typeof ceramic === 'string' ? new CeramicClient(ceramic) : ceramic
```

Comment:
@PaulLeCam , I changed this, because for some reason when I was calling this with `Command`'s `cermic` property which as far as I can say is a `CeramicClient`, the `instanceof` check didn't work. Are you ok with this `typeof` check that I used instead?

Reply:
If the `instanceof` check doesn't work it might be the symptom of a larger issue like having 2 different version of the `CeramicClient` class, which is bad. Can you investigate why this is happening please? Could it be related to the behavior of automatically installing peer dependencies?

Reply:
Sure, will do :)

Reply:
Ok, @PaulLeCam , the problem was just that the version of `@ceramicnetwork/http-client` was `2.2.1` in `devtools-node` and `2.3.0` in `cli`. My silly mistake, sorry. It's fixed now with the `instanceof` check.

Reply:
Can you revert to the `instanceof` check then so this kind of issue would continue to get flagged?